ORK1Kit/ORK1Kit/Common/ORK1TableContainerView.m

```diff
@@ -150,7 +150,7 @@ - (void)layoutSubviews {
         CGSize footerSize = [_continueSkipContainerView systemLayoutSizeFittingSize:(CGSize){_tableView.bounds.size.width,0} withHorizontalFittingPriority:UILayoutPriorityRequired verticalFittingPriority:UILayoutPriorityFittingSizeLevel];
         CGRect footerBounds = (CGRect){{0,0},footerSize};
         
-        CGFloat boundsHeightUnused = _tableView.bounds.size.height - _tableView.contentSize.height;
+        CGFloat boundsHeightUnused = _tableView.bounds.size.height - _tableView.contentSize.height - _tableView.adjustedContentInset.bottom - _tableView.adjustedContentInset.top;
```

Comment:
This code calculates the vertical distance from the table view content to the bottom of the tableview. And then uses that value as the footer height so the next button is pinned to the bottom. However it did not adjust for iPhone X's home bar. Change fixes that